GeneralStateTests/stZeroKnowledge/ecpairing_one_point_with_g2_zero_and_g1_invalid.json

```diff
@@ -0,0 +1,364 @@
+{
+    "ecpairing_one_point_with_g2_zero_and_g1_invalid" : {
+        "_info" : {
+            "comment" : "",
```

Comment:
this field should be filled by testeth if there is one in a filler file

Reply:
testeth does not do that.  So testeth has a bug.

Reply:
https://github.com/ethereum/cpp-ethereum/issues/4454